### ISSUE TITLE
Specifying steps along with log value in tflog

### ISSUE THREAD

**Issue by doantientai** [mrahtz/easy-tf-log#1]
Thank you for the wonderful tool!

I am trying to plot a validation loss on the same graph as training loss. The training loss is updated after 100 steps while validation loss is updated every 1000 steps, so... Is there any way to specify the global step when we log the value? Something like... tflog(key='loss', value="x", step=globlal_step)

**Comment by doantientai** [comment]
I think I found out a way around below, but now sure if it is correct:

```
def logkv(self, k, v, s):
    def summary_val(k, v):
        kwargs = {'tag': k, 'simple_value': float(v)}
        return tf.Summary.Value(**kwargs)
    summary = tf.Summary(value=[summary_val(k, v)])
    event = event_pb2.Event(wall_time=time.time(), summary=summary)
    # Use a separate step counter for each key
    if k not in self.key_steps:
        self.key_steps[k] = 0
    event.step = s
    self.writer.WriteEvent(event)
    self.writer.Flush()
    self.key_steps[k] = s
```

**Comment by mrahtz** [comment]
Yeah, that looks reasonable. I'll try to find time in the next week to implement this more cleanly, but very happy to accept a pull request in the meantime.

**Comment by mrahtz** [comment]
OK, check out v1.7. `tflog` now has a `step` argument.
